07-js_html_css_dom/portfolio/index.html

```diff
@@ -0,0 +1,59 @@
+    <meta charset="UTF-8">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="default.css">
+</head>
+<body>
+<section id="main">
+
+    <section id="content">
+        <section id="left">
+            <h1>My Portfolio</h1>
+            <p>Welcome to my portfolio!</p>
+            <h2>My Work</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Pellentesque elit eget gravida cum sociis. Dignissim diam quis enim lobortis
+                scelerisque fermentum. Ac tortor vitae purus faucibus ornare suspendisse sed. Arcu non sodales neque
+                sodales ut etiam. Ut tristique et egestas quis ipsum suspendisse ultrices. At quis risus sed vulputate
+                odio ut enim blandit. Mattis aliquam faucibus purus in massa tempor nec. Malesuada nunc vel risus
+                commodo viverra maecenas accumsan lacus. Pellentesque elit ullamcorper dignissim cras tincidunt lobortis
+                feugiat vivamus at. Ornare quam viverra orci sagittis eu. Et malesuada fames ac turpis egestas integer
+                eget aliquet. Magna eget est lorem ipsum. Mauris rhoncus aenean vel elit. Urna molestie at elementum eu
+                facilisis. Posuere sollicitudin aliquam ultrices sagittis orci a scelerisque purus semper. Tellus cras
+                adipiscing enim eu turpis egestas. Ut ornare lectus sit amet est placerat in. Facilisi nullam vehicula
+                ipsum a arcu cursus vitae congue. A pellentesque sit amet porttitor eget.</p>
+            <h2>My Skills</h2>
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et
+                dolore magna aliqua. Pellentesque elit eget gravida cum sociis. Dignissim diam quis enim lobortis
+                scelerisque fermentum. Ac tortor vitae purus faucibus ornare suspendisse sed. Arcu non sodales neque
+                sodales ut etiam. Ut tristique et egestas quis ipsum suspendisse ultrices. At quis risus sed vulputate
+                odio ut enim blandit. Mattis aliquam faucibus purus in massa tempor nec. Malesuada nunc vel risus
+                commodo viverra maecenas accumsan lacus. Pellentesque elit ullamcorper dignissim cras tincidunt lobortis
+                feugiat vivamus at. Ornare quam viverra orci sagittis eu. Et malesuada fames ac turpis egestas integer
+                eget aliquet. Magna eget est lorem ipsum. Mauris rhoncus aenean vel elit. Urna molestie at elementum eu
+                facilisis. Posuere sollicitudin aliquam ultrices sagittis orci a scelerisque purus semper. Tellus cras
+                adipiscing enim eu turpis egestas. Ut ornare lectus sit amet est placerat in. Facilisi nullam vehicula
+                ipsum a arcu cursus vitae congue. A pellentesque sit amet porttitor eget.</p>
+        </section>
+
+        <section id="right">
+            <h1>Side Bar</h1>
```

Comment:
This is an HTML nitpick: You've already got an `h1` in your document. An HTML document should in theory only have one `h1`.

Reply:
Ah, ok! And the <section> is new for me. I mostly just see people using <div>, so are sections exactly what they sound like? Just another way to organize content?